### DISCUSSION TITLE
How to use TTS

### DISCUSSION THREAD

**Discussion by Michael7575** [innightwolfsleep/llm_telegram_bot#83] [Q&A]
How do you use request an audio reply from the bot?

**Comment by innightwolfsleep** [comment]
![image](https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/assets/20921671/209c4847-cfc9-4067-a16e-e9c6a0b9e292)

Just chose voice with buttons)

**Reply by Michael7575** [reply]
<img width="1126" alt="image" src="https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/assets/54969137/34cca1e7-7ea7-41a5-b46f-705c406304cc">
I am getting empty audio files.

**Reply by Michael7575** [reply]
it works for me in the web ui

**Comment by Michael7575** [comment]
reinstalled webui and extension.
